integration-tests/downscope/downscope_test.go

```diff
@@ -0,0 +1,153 @@
+// Copyright 2021 Google LLC.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package downscope
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"testing"
+	"time"
+
+	"google.golang.org/api/option"
+
+	"cloud.google.com/go/storage"
```

Comment:
This module does not currently rely on this module and I think it is best to keep it that way. Can you refactor this test to use `https://pkg.go.dev/google.golang.org/api/storage/v1`. This is what this client uses under the hood.